crates/inline_completion_button/src/inline_completion_button.rs

```diff
@@ -258,7 +252,7 @@ impl Render for InlineCompletionButton {
                     return div().child(
                         IconButton::new("zed-predict-pending-button", zeta_icon)
                             .shape(IconButtonShape::Square)
-                            .indicator(Indicator::dot().color(Color::Error))
+                            .indicator(Indicator::dot().color(Color::Muted))
```

Comment:
I changed this color because the red was a little too distracting if we want to have it available from first launch.